docs/repos.md

```diff
@@ -4,77 +4,116 @@
-    React.js and webpack. The client is used to explore and manage content within GraphKB.
+## ETL / Data Loading
 
-- ## [GraphKB Python Adaptor](https://github.com/bcgsc/pori_graphkb_python)
+The GraphKB project also includes a loaders package which are used to import content from external knowledge bases and ontologies into GraphKB. Writing these loaders requires a strong understanding of the knowledge graph structure of GraphKB as well as the structure of the target resource. The loaders are written in javscript to be able to leverage the parser and schema JS packages used by the API and client. They are run with [NodeJS](https://nodejs.org/en/). A list of the popular supported inputs can be found in the [loading data section](./graphkb/loading_data.md).
+
+<div class='projects' markdown='1'>
+
+- ### [GraphKB Data Loaders](https://github.com/bcgsc/pori_graphkb_loader)
+
+    ![graphkb loaders](./images/graph-icon_outline.svg)
+
+    GraphKB loaders is responsible for all data import into GraphKB. Automatic Import modules are provided for a variety of external ontologies and knowledgebases such as: Ensembl, Entrez Genes, RefSeq, HGNC, Disease Ontology, NCI Thesaurus, CIViC, DoCM, etc.
+
+</div>
+
+## Python Adaptors
+
+The popularity of python in bioinformatics makes it one of the top choices for adaptors. These adaptors are written to help users integrate PORI into their existing bioinformatic workflows. They are published and installed via pip.
+
+```bash
+pip install graphkb ipr
+```
+
+A developer reference for both packages including the function signatures and package details can be found in the developer reference section [here](../developer_reference).
+
+<div class='projects' markdown='1'>
+
+- ### [GraphKB Python Adaptor](https://github.com/bcgsc/pori_graphkb_python)
 
     ![graphkb python](./images/graph-icon_outline.svg)
 
     Python adapter package for querying the GraphKB API. See the related
     [user manual](../graphkb/python/docs) for instructions on incorporating
     this into custom scripts.
 
-- ## [IPR API](https://github.com/bcgsc/pori_ipr_api)
-
-    ![ipr api](https://www.bcgsc.ca/gsc-logos/icon/logo-circle.png)
-
-    The Integrated Pipeline Reports (IPR) REST API manages data access to the IPR database. The API is
-    responsible for storing and server all data for reports.
+- ### [IPR Python Adaptor](https://github.com/bcgsc/pori_ipr_python)
 
-- ## [IPR client](https://github.com/bcgsc/pori_ipr_client)
+    ![ipr python](./images/wrench.svg)
 
-    ![ipr client](./images/pori-ipr-main-report-page.png)
+    Python adapter for generating reports uploaded to the IPR API. This python tool takes in variant inputs as tab-delimited files and annotates them using GraphKB. The resulting output is uploaded to IPR as a report. Additional report content such as images and metadata can be passed to be included in the report upload.
```

Comment:
'adapter' -> 'adaptor', i think

Reply:
I had to look this up! Turns out they are interchangeable but adapter is the more common variant so I will leave this one